example/mysite/docker_settings.py

```diff
@@ -1,28 +1,47 @@
+# An example settings file for deploying a Django app in a Docker container.
+# Uses environment variables to configure the majority of settings. This
+# pattern is sometimes attributed to the '12factor' app guidelines:
+# https://12factor.net/config
+
 from __future__ import absolute_import
 
 import environ
```

Comment:
Perhaps we need a comment here explaining that django-environ will need to be added to install_requires if adopting this settings file pattern.